integration_test/viss/features/viss_v2_multiple_paths.feature

```diff
@@ -20,19 +20,19 @@ Feature: VISS v2 Compliance Testing - Multiple Paths
   @ShouldHave
   Scenario: Request for multiple values from a single node
     # This scenario can be expanded based on specific use cases.
-    When I send a read request with path "Vehicle.Cabin.TemperatureSetpoint"
-    Then I should receive multiple values from a single node
+    When I search "Vehicle.Cabin" using a path filter "Door.*.*.IsOpen"
```

Comment:
General comment I dont know if the phrasing with `I` is the best. Who is I? Shouldnt it be When a client searches for .... Then the client should receive ...